index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <meta name="author" content="Mavericks Balitaan" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="headerText">Welcome</div>
+          <div class="menu">
+            <img src="./svg/hamburger.svg" alt="menu" />
+          </div>
+        </div>
+      </header>
+      <div class="headlineSection">
+        <div class="headlineTextGroup">
+          <div class="headlineText">Hey there. I'm Mavericks</div>
+          <div class="headlineText2">I'm a software developer</div>
+        </div>
+        <div class="supportText">
+          I can help you build a product ,<br />
+          feature or website Look through <br />
+          some of my work and experience! <br />
+          If you like what you see and have <br />
+          a project you need coded, <br />
+          don't hesitate to contact me.
+        </div>
+      </div>
+      <div class="iconBar">
+        <img src="./svg/twitter.svg" alt="twitter" />
+        <img src="./svg/linkedin.svg" alt="linkedin" />
+        <img src="./svg/medium.svg" alt="medium" />
+        <img src="./svg/github.svg" alt="github" />
+        <img src="./svg/angellist.svg" alt="angellist" />
+      </div>
```

Comment:
- [ ] Kindly move this part to inside the `headline` section because it's related to it.
- [ ] Kindly use `ul` instead of `div`, and wrap each image inside `li` tag and `a`, to link to your accounts 
Like
```
<ul>
  <li><a href=""><img ..../><a/></li>
  <li><a href=""><img ..../><a/></li>
  <li><a href=""><img ..../><a/></li>
  ............
</ul>
```